test/client/nodes_test.rb

```diff
@@ -41,7 +41,7 @@
   it "gets the hot threads for the node(s)" do
     str = $client.nodes.hot_threads :read_timeout => 2
     assert_instance_of String, str
-    assert_match %r/cpu usage by thread/, str
+    assert_match %r/:::/, str
```

Comment:
For hot_threads specifically, I'm not too worried about test correctness. It's an admin endpoint that's only useful for human viewers. I'd be willing to drop its tests if they're causing problems.